cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -5,46 +5,488 @@
+            }
+
+            &-link {
+                padding-top: @grid_gutter-width / 2;
+                padding-bottom: @grid_gutter-width / 2;
+            }
+        }
+    } );
+
+    // Desktop sizes.
     .respond-to-min( @bp-med-min, {
-        ul {
-            padding-left: 0;
-            margin-bottom: 0;
+        margin-right: -@grid_gutter-width / 2;
+        margin-left: -@grid_gutter-width / 2;
 
-            li {
+        &_trigger {
+            display: none;
+        }
+
+        &_content {
+            &-list {
+                padding-left: 0;
                 margin-bottom: 0;
+            }
+
+            &-item {
+                margin-right: 0.25em;
+                display: inline-block;
 
                 &:first-child {
                     display: none;
                 }
             }
+
+            &-link {
+                padding-bottom: @grid_gutter-width;
+                margin-right: @grid_gutter-width / 2;
+                margin-left: @grid_gutter-width / 2;
+
+                font-size: unit( 18px / @base-font-size-px, em );
+
+                &:first-child {
+                    margin-left: 0;
+                }
+
+                // TODO: Show menu on hover and remove :hover selector.
+                &[aria-expanded="true"],
+                &__current,
+                &:hover {
+                    padding-bottom: @grid_gutter-width - 6px;
+                    border-bottom: 6px solid @black;
+                }
+            }
+        }
+
+        &_subcontent {
+            overflow: hidden;
+            position: absolute;
+            left: 0;
+            z-index: 10;
+            width: 100%;
+
+            // Set an arbtrary min-height to create a "window" into which
+            // the menu will slide into view.
+            min-height: 600px;
+            pointer-events: none;
+
+            &-wrapper {
+                pointer-events: auto;
+                background-color: @gray-5;
+                border-top: 1px solid @gray-50;
+                border-bottom: 1px solid @gray-50;
+                transform: translateY( -100% );
+                transition: transform 0.25s ease-out;
+
+                .u-drop-shadow-after();
+
+                .wrapper {
+                    padding: @grid_gutter-width;
+                }
+            }
+
+            &[aria-expanded="true"] {
+
+                .o-mega-menu_subcontent-wrapper {
+                    display: block;
+                    transform: translateY( 0 );
+                }
+            }
+
+            &-btn__back {
+                display: none;
+            }
+
+            &-title {
+                padding-bottom: @grid_gutter-width / 2;
+                margin-bottom: unit( @grid_gutter-width / 2 / 18px, em );
+
+                font-size: 1em;
+            }
+
+            &-list {
+                .grid_column(4);
+
+                &:first-child {
+                    border-left: 0;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/c2ce340878e2904d023b2e075cfd8d61e4598347